lib/CL/devices/pthread/pthread.c

```diff
@@ -71,7 +71,6 @@ struct data {
 void
 pocl_pthread_init_device_ops(struct pocl_device_ops *ops)
 {
-  pocl_ventus_init_device_ops(ops);
```

Comment:
Removed redundant code to prevent pocl_ventus_init from being called twice. Please put a separate commit.

Reply:
“extensions" is recommended.